peers/app_request_network.go

```diff
@@ -81,6 +81,7 @@ func NewNetwork(
 	trackedSubnets set.Set[ids.ID],
 	manuallyTrackedPeers []info.Peer,
 	cfg Config,
+	allowPrivateIPs bool,
```

Comment:
We can add a `GetAllowPrivateIPs` method to the `peers.Config` interface since it's present in both relayer and signature aggregator configs. That way we don't have to pass in this param.

Reply:
done